### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by mitermayer**
Fixing a bunch of typos

### CODE REVIEW

File: website/static/lib/parser-postcss.js [hunk 1]
```diff
@@ -23389,7 +23389,7 @@ class TokenizeError extends Error {
     super(message);
 
     this.name = this.constructor.name;
-    this.message = message || 'An error ocurred while tokzenizing.';
+    this.message = message || 'An error occurred while tokzenizing.';
```
Comment (duailibe) [inline website/static/lib/parser-postcss.js]: missed one here

**Comment by mitermayer** [comment]
Reverted back changes on the static libs

File: tests/css_less_docs/less_docs.css [hunk 2]
```diff
@@ -1076,7 +1076,7 @@ div {
   .mixin();
 }
 
-@size: globaly-defined-value; // callers parent scope - no protection
+@size: globally-defined-value; // callers parent scope - no protection
```
Comment (azz) [inline tests/css_less_docs/less_docs.css]: Is there a snapshot for this file?

Comment (mitermayer) [reply tests/css_less_docs/less_docs.css]: There is no file with that name, also all tests are passing

Comment (azz) [reply tests/css_less_docs/less_docs.css]: Is there a jsfmt.spec.js in that directory (mobile atm)

Comment (mitermayer) [reply tests/css_less_docs/less_docs.css]: there is no __snapshots__ folder in that directory `tests/css_less_docs`

Comment (azz) [reply tests/css_less_docs/less_docs.css]: Uh-oh. Raised #3214

**Pull request merged**